src/engraving/compat/midi/compatmidirender.cpp

```diff
@@ -1146,4 +1146,9 @@ int CompatMidiRender::slideTicks(Chord* chord)
 
     return SLIDE_DURATION;
 }
+
+int CompatMidiRender::tick(const CompatMidiRendererInternal::Context& ctx, int tick)
```

Comment:
Name **tick** is a bit confusing.
It is returning the pause tick. I think something like **pauseTick** is less confusing

Reply:
> Or it is actually returning not only pause tick. Does it used only for pauses or not?
depends if pause map is filled, then it will return tick with offset